src/Operation/Count.php

```diff
@@ -162,12 +162,27 @@ public function execute(Server $server)
         return (integer) $result->n;
     }
 
+    public function getCommandDocument()
+    {
+        return $this->createCommandDocument();
```

Comment:
I was originally going to suggest factoring out the private `createCommandDocument()` method and just turning it into this public method, but I think it's reasonable to keep them separate for now. That will allow the interface to be tweaked without disturbing the innards of each operation class.

I'm contemplating whether it may make sense to have some form of "explain" in the interface method name, but we'll have plenty of time to debate names before releasing 1.4.0. This is totally fine as-is :)